Package.swift

```diff
@@ -14,22 +14,52 @@ import PackageDescription
 let package = Package(
   name: "SwiftProtobuf",
   products: [
-    .executable(name: "protoc-gen-swift", targets: ["protoc-gen-swift"]),
-    .library(name: "SwiftProtobuf", targets: ["SwiftProtobuf"]),
-    .library(name: "SwiftProtobufPluginLibrary", targets: ["SwiftProtobufPluginLibrary"]),
+    .executable(
+        name: "protoc-gen-swift",
+        targets: ["protoc-gen-swift"]
+    ),
+    .library(
+        name: "SwiftProtobuf",
+        targets: ["SwiftProtobuf"]
+    ),
+    .library(
+        name: "SwiftProtobufPluginLibrary",
+        targets: ["SwiftProtobufPluginLibrary"]
+    ),
+    .plugin(
```

Comment:
Apart from the reformatting, you've added one product `.plugin`

Reply:
Yes we are just vending a new `plugin` product